docs/entities/Listing.md

```diff
@@ -1,4 +1,8 @@
+<<<<<<<< HEAD:docs/entities/Listing.md
 # Listing
+========
+# Product
+>>>>>>>> 86d9cb3d4fb487087b331d19e9440e3ed13cc596:docs/entities/Product.md
```

Comment:
Revert this commit because this means there was a merge conflict that needed to be resolved and you forced the commit without resolving it first.